packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -73,6 +73,15 @@ export class OpGroupingManager {
 
 	public ungroupOp(op: ISequencedDocumentMessage): ISequencedDocumentMessage[] {
 		if (!isGroupContents(op.contents)) {
+			// Align the worlds of what clientSequenceNumber represents when grouped batching is enabled
+			if (this.groupedBatchingEnabled) {
+				return [
+					{
+						...op,
+						clientSequenceNumber: 1,
```

Comment:
#nit.  I would  move `let fakeCsn = 1;` above and use `fakeCsn` here. It sends a more explicit message IMO.

Also, can we have a unit test for this behavior?